core/arch/arm/kernel/sub.mk

```diff
@@ -33,7 +33,7 @@ srcs-y += wait_queue.c
 srcs-$(CFG_PM_STUBS) += pm_stubs.c
 
 srcs-$(CFG_GENERIC_BOOT) += generic_boot.c
-srcs-$(CFG_GENERIC_BOOT) += generic_core_bootcfg.c
+srcs-y += generic_core_bootcfg.c
```

Comment:
This file is removed (content moved into `core/arch/arm/mm/core_mmu.c`) in a later patch from https://github.com/jenswi-linaro/optee_os/tree/nlin_map